src/sentry/tasks/commit_context.py

```diff
@@ -142,15 +143,14 @@ def queue_comment_task_if_needed(
     bind=True,
 )
 def process_commit_context(
-    self,
-    event_id,
-    event_platform,
-    event_frames,
-    group_id,
-    project_id,
-    sdk_name=None,
-    **kwargs,
-):
+    self: Task,
+    event_id: str,
+    event_platform: str,
+    event_frames: Sequence[Mapping[str, Any]],
```

Comment:
```suggestion
    event_frames: Sequence[Mapping[str, object]],
```

This may or may not work but it is preferred.

Reply:
I'm matching the type where this comes from which is https://github.com/getsentry/sentry/blob/ae09a3b1fde02a6fd231e1a9a80d2b06707fa6c0/src/sentry/utils/event_frames.py#L176

Would love if that response was typed better 😞

Reply:
Feel free to go ahead as-is.